### ISSUE TITLE
As PhO I want to have data adjusted in the console

### ISSUE THREAD

**Issue by evaklopp** [buzzn/scrum#53]
**PR**
---
**Story**
AS PhO I checked data in the console and found some mistakes, which I want to have adjusted in the console.

**DESCRIPTION** 
![buzzn com - register](https://user-images.githubusercontent.com/22292464/33264284-310d71c6-d36d-11e7-8b51-6d7af0a64381.jpg)
 
**DoD**

- [ ] OBIS becomes //waiting for answer//
- [ ] Metering Point: please add "ID"

Readings: 
- [ ] change unit from Wh to kWh and adjust the number of  "0" accordingly

**Comment by evaklopp** [comment]
obsolete ticket, just talked to PhO

**Comment by mkristian** [comment]
I am not sure if PhO is the right user for the Console, for example we store the readings in their base units `Wh` or `W` or `m^3` in the DB and the Console is primarily to display the data from the DB. in the admin-app (not sure how to call it right now) we convert and display the data as mentioned in this issue. Dito the OBIS is what we call the field in the DB.

Next there is this 'type' which indicates it is RealMeter and VirtualMeter will have type `virtual` and as such have a totally set of 'relevant' info, i.e. the obis and other fields will not be used. so there is obvious mapping from this to what we call `RealMeter` in the admin-app.

In short I am afraid that the console is more confusing then helpful for PhO

**Comment by mkristian** [comment]
OK I see . . . good !

**Comment by evaklopp** [comment]
I had the same feelings and thoughts on it as you @mkristian. I asked PhO last week to help with prioritizing, that´s why he has seen this screenshot. While you wrote your comment, I just talked to him and decided that it is better to have him only on the Fe/staging view. He is not a BE person at all. So, no worries, ticket is obsolet.
